src/vma/util/lock_wrapper.h

```diff
@@ -68,7 +68,7 @@ class lock_base
 #endif
 protected:
 	tscval_t                start_lock_wait() { return 0; };
-	void                    end_lock_wait(tscval_t start_time) {start_time = 0; NOT_IN_USE(start_time);};
+	void                    end_lock_wait(tscval_t start_time) {start_time = 0; start_time = start_time;};
```

Comment:
What risk do you see?
using 
`start_time = start_time`
instead of 
`(void) start_time`
should be safe enough. I guess that most compilers optimize both variants removing from result binary.